docs/source/development-guidelines.rst

```diff
@@ -0,0 +1,223 @@
+
+
+Packaging
+---------
+To learn about how we build Zipline packages on `Anaconda`__ you can read `this`__ section in our release process notes.
+
+__ https://anaconda.org/
+__ http://www.zipline.io/release-process.html#uploading-conda-packages
+   
+Contributing to the Docs
+------------------------
+
+If you'd like to contribute to the documentation, you can navigate to ``docs/source/`` where each `reStructuredText`__ or ``.rst``, file is a separate section here on zipline.io. To add a section, create a new file called ``some-descriptive-name.rst`` and add ``some-descriptive-name`` to ``appendix.rst``. To edit a section, simply open up one of the existing files, make your changes, and save them.
+
+__ https://en.wikipedia.org/wiki/ReStructuredText
+
+We use `Sphinx`__ to generate documentation for Zipline, which you will need to install by running:
+
+__ http://www.sphinx-doc.org/en/stable/
+
+
+.. code-block:: bash
+
+   $ pip install -r ./etc/requirements_docs.txt
+
+To build and view the docs locally, run:
+
+.. code-block:: bash
+
+   # assuming you're in the Zipline root directory
+   $ cd docs
+   $ make html
+   $ {BROWSER} build/html/index.html
+
+
+Commit messages
+---------------
+
+Standard prefixes to start a commit message:
+
+.. code-block:: bash
+
+   BLD: change related to building Zipline
+   BUG: bug fix
+   DEP: deprecate something, or remove a deprecated object
+   DEV: development tool or utility
+   DOC: documentation
+   ENH: enhancement
+   MAINT: maintenance commit (refactoring, typos, etc)
+   REV: revert an earlier commit
+   STY: style fix (whitespace, PEP8, flake8, etc)
+   TST: addition or modification of tests
+   REL: related to releasing Zipline
+   PERF: performance enhancements
+
+
+Some commit style guidelines:
+
+Commit lines should be no longer than `72 characters`__. The first line of the commit should include one of the above prefixes. There should be an empty line between the commit subject and the body of the commit. In general, the message should be in the imperative tense. Best practice is to include not only what the change is, but why the change was made.
+
+__ https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project
+
+e.g.
+
+.. code-block:: text
+
+   MAINT: Remove unused calculations of max_leverage, et al.
+
+   In the performance period the max_leverage, max_capital_used,
+   cumulative_capital_used were calculated but not used.
+
+   At least one of those calculations, max_leverage, was causing a
+   divide by zero error.
+   
+   Instead of papering over that error, the entire calculation was
+   a bit suspect so removing, with possibility of adding it back in
+   later with handling the case (or raising appropriate errors) when
+   the algorithm has little cash on hand.
+
+
+Formatting Docstrings
+---------------------
+
+When adding or editing docstrings for classes, functions, etc, we use the numpy `HOWTO_DOCUMENT`__ file as the canonical reference.
```

Comment:
What do you think about "we use numpy as the canonical reference" where "numpy" links to the HOWTO?